pkg/porter/create.go

```diff
@@ -4,40 +4,80 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create function creates a porter configuration in the specified directory or in the current directory if no directory is specified.
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// Use the current directory if no directory is passed
+	if bundleName == "" {
+		bundleName = p.FileSystem.Getwd()
+	}
+
+	// Check if the directory in which bundle needs to be created already exists.
+	// If not, create the directory.
+	_, err := os.Stat(bundleName)
+	if err != nil {
+		if os.IsNotExist(err) {
+			// This code here attempts to create the directory in which bundle needs to be created,
+			// if the directory does not exist.
+			// bundleName can handle both the relative path and absolute path into consideration,
+			// For example if we want to create a bundle named mybundle in an existing directory /home/user we can call porter create /home/user/mybundle or porter create mybundle in the /home/user directory.
+			// If we are in a directory /home/user and we want to create mybundle in the directory /home/user/directory given the directory exists,
+			// we can call porter create directory/mybundle from the /home/user directory or with any relative paths' combinations that one can come up with.
+			// Only condition to use porter create with absolute and relative paths is that all the directories in the path except the last one should strictly exist.
+			err = os.Mkdir(bundleName, os.ModePerm)
```

Comment:
```suggestion
			err = os.MkdirAll(bundleName, os.ModePerm)
```
This will make sure that this creates bundles such as `porter create mybundle/newestbundle`